microsite/sidebars.js

```diff
@@ -489,34 +485,40 @@ module.exports = {
         ],
       },
     ],
-    'Designing for Backstage': [
-      'dls/design',
-      'dls/component-design-guidelines',
-      'dls/contributing-to-storybook',
-      'dls/figma',
-    ],
     Tutorials: [
-      'tutorials/quickstart-app-plugin',
-      'tutorials/react-router-stable-migration',
-      'tutorials/react18-migration',
-      'tutorials/package-role-migration',
-      'tutorials/migrating-away-from-core',
-      'tutorials/configuring-plugin-databases',
-      'tutorials/switching-sqlite-postgres',
-      'tutorials/using-backstage-proxy-within-plugin',
-      'tutorials/yarn-migration',
-      'tutorials/migrate-to-mui5',
-      'tutorials/auth-service-migration',
-      'tutorials/enable-public-entry',
-      'tutorials/setup-opentelemetry',
+      { 'Non-technical': ['overview/adopting'] },
+      {
+        Techical: [
+          'tutorials/quickstart-app-plugin',
+          'tutorials/react-router-stable-migration',
+          'tutorials/react18-migration',
+          'tutorials/package-role-migration',
+          'tutorials/migrating-away-from-core',
+          'tutorials/configuring-plugin-databases',
+          'tutorials/switching-sqlite-postgres',
+          'tutorials/using-backstage-proxy-within-plugin',
+          'tutorials/yarn-migration',
+          'tutorials/migrate-to-mui5',
+          'tutorials/auth-service-migration',
+          'tutorials/enable-public-entry',
+          'tutorials/setup-opentelemetry',
+          'accessibility/index',
+        ],
+      },
     ],
-
-    FAQ: ['faq/index', 'faq/product', 'faq/technical'],
-    Accessibility: ['accessibility/index'],
+    FAQ: ['faq/index', 'faq/technical', 'faq/product'],
     Contribute: [
       'contribute/index',
       'contribute/getting-involved',
       'contribute/project-structure',
+      {
+        'Designing for Backstage': [
+          'dls/design',
+          'dls/component-design-guidelines',
+          'dls/contributing-to-storybook',
+          'dls/figma',
```

Comment:
Maybe not for this PR but I think the `figma` page could basically just be a line somewhere in `design` as its literally just a URL 😅

Reply:
Yeah, I had the same thought as I was going through. I was thinking it would fit into the "rewriting the docs" issue we've been talking about, I don't think we have a concrete issue to link to yet for that. Let me add a note for the next SIG :)